index.js

```diff
@@ -65,6 +66,11 @@ render(
           </Route>
         </Route>
         <Route path="explorer" component={Explorer} />
+        <Route path="heroes" component={Heroes}>
+          <IndexRoute component={HeroesIndex} />
+          <Route path="ranking/:hero_id" component={Ranking} />
```

Comment:
for parity with existing site I'd like this to be rankings and benchmarks.

I'll merge this in anyway, but it should be fixed (either you can submit a PR or I can patch it myself)